README.md

```diff
@@ -38,7 +38,26 @@ elastic-package help
 
 ## Development
 
-Download and build the latest main of `elastic-package` binary:
+Even though the project is "go-gettable", there is the [`Makefile`](./Makefile) present, which can be used to build,
+install, format the source code among others. Some examples of the available targets are:
+
+`make build` - build the tool source
+
+`make clean` - delete elastic-package binary and build folder
+
+`make format` - format the Go code
+
+`make check` - one-liner, used by CI to verify if source code is ready to be pushed to the repository
+
+`make install` - build the tool source and move binary to `$GOBIN`
+
+`make gomod` - ensure go.mod and go.sum are up to date
+
+`make update` - update README.md file
+
+`make licenser` - add the Elastic license header in the source code
```

Comment:
Added a few more targets, and change `make vendor` target by the `make gomod` one.